### PR TITLE
Added functionality of creating and saving new task to db

### PR DESCRIPTION
**Pull request by MrSancho42**

### CODE REVIEW

**Review by s0fko — APPROVED** [review]

File: codeArena/task/serializers.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    class Meta:
+        model = Task
+        exclude = ['_id', 'created_at', 'updated_at']
+
+    def validate_languages(self, value):
+        """
+        Checks for languages in the database.
+        """
+
+        language_list = [language.name
```
Comment (vvelizariy) [inline codeArena/task/serializers.py]: WIll be more simple Language.objects.values_list('name', flat=True)

**Review by MetalMari — APPROVED** [review]

File: codeArena/task/serializers.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    def validate_categories(self, value):
+        """
+        Checks for categories in the database.
+        """
+
+        category_list = [category.name
+                         for category in Category.objects.all()]
+
+        if all(category in category_list for category in value):
```
Comment (vvelizariy) [inline codeArena/task/serializers.py]: Would be more obvious that you are checking for difference from input value with existing languages
```
if list(set(value) - set(category_list)):
    return value
```

Comment (MrSancho42) [reply codeArena/task/serializers.py]: It`s not working)

Comment (MrSancho42) [reply codeArena/task/serializers.py]: I resolve it like this:

`
if set(value).issubset(set(category_list)):
    return value
`

**Review by s0fko — APPROVED** [review]

**Review by Viktorika-p — APPROVED** [review]

**Pull request merged**